apicast/src/configuration.lua

```diff
@@ -143,13 +143,16 @@ function _M.parse_service(service)
       hosts = proxy.hosts or { 'localhost' }, -- TODO: verify localhost is good default
       api_backend = proxy.api_backend,
       error_auth_failed = proxy.error_auth_failed,
+      error_limits_exceeded = proxy.error_limits_exceeded,
```

Comment:
We probably should start providing default values for this. Especially when system is behind with the UI.

Reply:
Agreed. I added defaults based on the ones shown in the UI.
This is included in the latest 3 commits: implementation + integration tests + unit tests.